docs/user-guides/retrieval.md

```diff
@@ -0,0 +1,191 @@
+# Retrieval in CLIP-as-service
+`CLIP-as-service` offers us high quality embeddings. Retrieval is one of the most common use cases for embeddings. Retrieval in CLIP-as-service can support indexing a very large dataset(millions/billions) and querying within several million seconds.
```

Comment:
milliseconds

Reply:
good catch

Reply:
`several millionseconds` is not clear. 10 ms, vs 100ms, vs 500ms. As my understanding, 10ms is serveral ms. I don't think we can archive this performance.